GameRoom.ts

```diff
@@ -84,8 +84,6 @@ export class GameRoom extends Room<GameState> {
         
         const metadata = this.metadata as {roomName, author, skin, randomizeTiles};
 
-        console.log(this.metadata, metadata.roomName, metadata.author, metadata.skin, metadata.randomizeTiles);
```

Comment:
Next time clean your commit structure. Otherwise this will lead to a longer reviewing prociess and a confused college. Please refer to company guidelines.